### PR TITLE
Add uiMetadata to viewV2

### PR DESCRIPTION
**Pull request by aptkingston**
## Description
Tiny PR to add a UI metadata property to views. This is needed for the new view calculations feature. It's totally opt-in and is untouched by the server so is completely safe to merge into master without feature flags.

### CODE REVIEW

File: packages/server/src/api/routes/tests/viewV2.spec.ts [hunk 1]
```diff
@@ -177,6 +177,9 @@ describe.each([
               visible: true,
             },
           },
+          uiMetadata: {
+            foo: "bar",
+          },
```
Comment (samwho) [inline packages/server/src/api/routes/tests/viewV2.spec.ts:182]: You probably want at least one assertion somewhere to make sure we do actually return this when fetching views from the API.

Comment (aptkingston) [reply packages/server/src/api/routes/tests/viewV2.spec.ts:182]: That's what the update test does already isn't it? It's asserting that the response from the update call matches the payload sent up, which includes the `uiMetadata` field that should have been saved.

Comment (samwho) [reply packages/server/src/api/routes/tests/viewV2.spec.ts:182]: Oh I completely missed that, sorry!

**Review by samwho — APPROVED** [review]

**Review by adrinr — APPROVED** [review]
Nice to see this ui metadata not directly added in the schema :)

**Pull request merged**